profiles/base.yaml

```diff
@@ -193,6 +193,7 @@ crossing_training_requires_movement:
 - Trading
 forage_item:
 braid_item:
+collect_item: rock
```

Comment:
I don't think the name of this setting makes it clear that it is only used in `combat-trainer` (nor does its placement next to some non-combat settings).